application/android/app/src/main/java/com/group12/beabee/InputValidator_ihsan.java

```diff
@@ -0,0 +1,20 @@
+package com.group12.beabee;
+
+public class InputValidator_ihsan {
+
+    public static boolean IsSearchQueryNonNull(String input){
+        if (input==null)
+            return false;
+
+        return input.length() >= 0;
+    }
+
+    public static boolean IsQueryAtLeastOneCharacterLength(String input){
+        if (input==null)
+            return false;
+
+        return input.length() >= 1;
+    }
+
```

Comment:
Nice unittest it can be used both in user search and also in market place search